### PR TITLE
[14.0][MIG] website_snippet_country_dropdown: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT32147

Main changes:
- I used the image_url field of res.country to show the flags. The way it was done in v13 doesn't work in v14 because now there are no image fields in res.country model.

### CODE REVIEW

File: website_snippet_country_dropdown/i18n/en.po [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+#~ msgstr "Country Dropdown Options"
+
+#~ msgid "Field name value"
+#~ msgstr "Field name value"
+
+#~ msgid "Helper!"
+#~ msgstr "Helper!"
+
+#~ msgid "Set input options"
+#~ msgstr "Set input options"
```
Comment (CarlosRoca13) [inline website_snippet_country_dropdown/i18n/en.po]:
```suggestion

```

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:  Just a minor comment

File: website_snippet_country_dropdown/static/src/js/website_snippet_country_dropdown.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        },
+        lazy_image_load: function () {
+            const _this = this;
+            this.$dropdown_list.children("a").each(function () {
+                const $elm = $(this);
+                const $img = $elm.children("img");
+                if (!$img.attr("src") && _this.is_option_visible($elm)) {
+                    $elm.children("img").attr("src", $elm.data("country_image_url"));
+                }
+            });
```
Comment (Tardo) [inline website_snippet_country_dropdown/static/src/js/website_snippet_country_dropdown.js]:
```suggestion
            this.$dropdown_list.children("a").each((index, item) => {
                const $elm = $(item);
                const $img = $elm.children("img");
                if (!$img.attr("src") && this.is_option_visible($elm)) {
                    $elm.children("img").attr("src", $elm.data("country_image_url"));
                }
            });
```

**Review by Tardo — APPROVED** [review]
LGTM, code review... only non-blocking comments.

**Comment by ernestotejeda** [comment]
Thanks @CarlosRoca13 @Tardo changes done

**Pull request merged**